### PR TITLE
Hotfix/1.47.1

### PR DESCRIPTION
**Pull request by anxolin**
# Summary

- [x] Allow to define custom RPC nodes (and change to NodeReal instead of Infura) https://github.com/cowprotocol/cowswap/pull/3160
- [x] Don't fetch tokens list ens hashes (#3152)
- [x] Approve order token from allowance warning tooltip (#3144)
- [x]  Approve infinite amount for TWAP (#3141)

# Test Plan
- [x] Add a test to trigger the allowance from pending limit/twap order 
- [x] SWAP in Gnosis Chain
- [x] SWAP in Goerli
- [x] SWAP in Mainnet
- [x] TWAP in Gnosis Chain (verify the approval is set to infinite).
- [x] TWAP in Goerli (verify the approval is set to infinite).
- [x] LIMIT ORDER in Gnosis Chain
- [x] LIMIT ORDER in Goerli
- [x] LIMIT ORDER in Mainnet

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -144,20 +144,24 @@ To set your own `AppData`, change `REACT_APP_FULL_APP_DATA_<environment>` enviro
...
-You can change the supported networks and their RPC endpoint.
-
-To have the interface default to a different network when a wallet is not connected:
-
-1. Change `REACT_APP_NETWORK_ID` to `"{YOUR_NETWORK_ID}"`. This will be your default network id
-2. Define your own list of supported networks:
+You should set your own RPC endpoints by defining the following environment variables:
 
 ```ini
-REACT_APP_SUPPORTED_CHAIN_IDS="1,100,5"
```
Comment (alfetopito) [inline README.md:155]: There are more places where this is still defined https://github.com/search?q=repo%3Acowprotocol%2Fcowswap+REACT_APP_SUPPORTED_CHAIN_IDS&type=code

**Review by alfetopito — APPROVED** [review]
Only checked the code, have not tested it

File: README.md (hunk 1 above)
Comment (anxolin) [reply README.md:155]: done in a follow up PR https://github.com/cowprotocol/cowswap/pull/3161/commits/37c6e4b33bb27da0761f8df6e2ac784157ea2ed4

**Review by shoom3301 — APPROVED** [review]

**Comment by anxolin** [comment]
Theres a small issue with the styles in the warning tooltip, other than that, i think this version is ready

image

**Pull request merged**